normal/gmail.profile

```diff
@@ -0,0 +1,83 @@
+# Make requests look like GMail web requests
+#
+# Author: @ChrisTruncer
+
+https-certificate {
+	set CN       "gmail.com";
+	set O        "Google's GMail";
+	set C        "US";
+	set L        "Mountain View";
+	set OU       "Google Mail";
+	set ST       "CA";
+	set validity "365";
+}
+
+http-get {
+	set uri "/mail/u/1/";
+	client {
+		header "Content-Type" "application/octet-stream";
```

Comment:
No need for client to send a Content-Type unless it's a POST request. This stands out as somewhat strange.